cmd/acra-connector/acra-connector.go

```diff
@@ -149,23 +149,23 @@ func main() {
 	keysDir := flag.String("keys_dir", keystore.DEFAULT_KEY_DIR_SHORT, "Folder from which will be loaded keys")
 	clientId := flag.String("client_id", "", "Client id")
 	acraHost := flag.String("acra_host", "", "IP or domain to acra daemon")
-	acraCommandsPort := flag.Int("acra_commands_port", cmd.DEFAULT_ACRA_API_PORT, "Port of acra http api")
+	acraCommandsPort := flag.Int("acra_commands_port", cmd.DEFAULT_ACRA_API_PORT, "Port of Acra HTTP api")
 	acraPort := flag.Int("acra_port", cmd.DEFAULT_ACRA_PORT, "Port of acra daemon")
```

Comment:
Probably, we can rename `cmd.DEFAULT_ACRA_PORT` → `cmd.DEFAULT_ACRASERVER_PORT` or `cmd.DEFAULT_SERVER_PORT` and so on?

Reply:
renamed constants, see in last commit

Reply:
we will rename arguments a bit later in a separate chunk of refactoring